cypress/integration/recipe_detail_spec.js

```diff
@@ -0,0 +1,432 @@
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="recipe-photo"]')
+      .should('have.prop', 'tagName')
+      .should('eq', 'IMG');
+
+    cy.get('[data-testid="recipe-title"]').contains('Spicy Arrabiata Penne');
+
+    cy.get('[data-testid="recipe-category"]').contains('Vegetarian');
+
+    cy.get('[data-testid="0-ingredient-name-and-measure"]').contains('penne rigate');
+    cy.get('[data-testid="0-ingredient-name-and-measure"]').contains('1 pound');
+    cy.get('[data-testid="1-ingredient-name-and-measure"]').contains('olive oil');
+    cy.get('[data-testid="1-ingredient-name-and-measure"]').contains('1/4 cup');
+    cy.get('[data-testid="2-ingredient-name-and-measure"]').contains('garlic');
+    cy.get('[data-testid="2-ingredient-name-and-measure"]').contains('3 cloves');
+    cy.get('[data-testid="3-ingredient-name-and-measure"]').contains('chopped tomatoes');
+    cy.get('[data-testid="3-ingredient-name-and-measure"]').contains('1 tin');
+    cy.get('[data-testid="4-ingredient-name-and-measure"]').contains('red chile flakes');
+    cy.get('[data-testid="4-ingredient-name-and-measure"]').contains('1/2 teaspoon');
+    cy.get('[data-testid="5-ingredient-name-and-measure"]').contains('italian seasoning');
+    cy.get('[data-testid="5-ingredient-name-and-measure"]').contains('1/2 teaspoon');
+    cy.get('[data-testid="6-ingredient-name-and-measure"]').contains('basil');
+    cy.get('[data-testid="6-ingredient-name-and-measure"]').contains('6 leaves');
+    cy.get('[data-testid="7-ingredient-name-and-measure"]').contains('Parmigiano-Reggiano');
+    cy.get('[data-testid="7-ingredient-name-and-measure"]').contains('spinkling');
+
+    cy.get('[data-testid="instructions"]').contains('Bring a large pot of water to a boil. Add kosher salt to the boiling water, then add the pasta. Cook according to the package instructions, about 9 minutes.\r\nIn a large skillet over medium-high heat, add the olive oil and heat until the oil starts to shimmer. Add the garlic and cook, stirring, until fragrant, 1 to 2 minutes. Add the chopped tomatoes, red chile flakes, Italian seasoning and salt and pepper to taste. Bring to a boil and cook for 5 minutes. Remove from the heat and add the chopped basil.\r\nDrain the pasta and add it to the sauce. Garnish with Parmigiano-Reggiano flakes and more basil and serve warm.');
+
+    cy.get('[data-testid="video"]').should('exist');
+
+    cy.get('[data-testid*="recomendation-card"]').should('exist');
+  });
+
+  it('verifica se os elementos existem na tela de detalhes de bebida', () => {
+    cy.visit('http://localhost:3000/bebidas/178319', {
+      onBeforeLoad(win) {
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="recipe-photo"]')
+      .should('have.prop', 'tagName')
+      .should('eq', 'IMG');
+
+    cy.get('[data-testid="recipe-title"]').contains('Aquamarine');
+
+    cy.get('[data-testid="recipe-category"]').contains('Cocktail');
```

Comment:
Para a tela de detalhes da bebida a `recipe-category` na verdade exibe se a bebida é alcoólica ou não, ao invés de usar a categoria propriamente dita, conforme escrito no requisito (**[...] a categoria (ou se é ou não alcoólico) [...]**).

Então aqui deveria ser:

```suggestion
    cy.get('[data-testid="recipe-category"]').contains('Alcoholic');
```